README.md

```diff
@@ -73,36 +73,18 @@ Write `yes` and press ENTER to establish a first connection.
 
 ### For people using Python on a cluster node
 
-- create a **local** python environment using conda or pipenv. You need to ensure your environment is on the `/scratch/`, not `/home/`, to do so check the guides just below.
+- create a **local** python environment using conda, virtualenv or pipenv. 
+- **Note**: Before creating environments, you need to create a $USER folder on `/scratch` (`mkdir /scratch/$USER` and replace `$USER` with your username). If you do not have the rights to create the folder on `/scratch`, consider creating it in `/scratch/students/` (`mkdir /scratch/students/$USER`). 
```

Comment:
Students don't have the right to make a folder on scratch, they can only do it in scratch/students. Be clear about this distinction.

Reply:
Keep in mind that this file is mostly for the students.

Reply:
https://github.com/dh-epfl-students/student-project-howtos/pull/4/commits/cf2a85e17410de3ef350bd34fd408dfe1adac539